charts/bookkeeper/values.yaml

```diff
@@ -57,7 +57,13 @@ storage:
 jvmOptions:
   memoryOpts: ["-Xms1g", "-XX:MaxDirectMemorySize=2g"]
   gcOpts: []
-  gcLoggingOpts: []
+  ## Unified JVM Logging is used for GC logging from Java 9
+  ## Hence use the following options if the bookkeeper version being deployed
+  ## is based on Java 9 or higher
+  # gcLoggingOpts: ["-Xlog:gc*,safepoint::time,level,tags:filecount=5,filesize=64m"]
+  ## However, if the bookkeeper version being deployed is based on a lower version
+  ## of Java (JDK 8 or below), we should be using the following options for GC logging instead
+  gcLoggingOpts: ["-XX:+PrintGCDetails","-XX:+PrintGCApplicationStoppedTime","-XX:+UseGCLogFileRotation","-XX:NumberOfGCLogFiles=5","-XX:GCLogFileSize=64m"]
```

Comment:
Why is this the default value? These are the JDK8 values, right? We should use by default the JDK11 ones, which are the default today in Bookkeeper and leave these ones commented.

Reply:
The bookkeeper version in this values.yaml is 0.7.0 which uses JDK 8
The default values can be changed once pravega/bookkeeper:0.9.0 has been released in open-source and can be used in this values.yaml file

Reply:
Pravega 0.9.0 is released already, I think that we need to set this by default for JDK11.